src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -21,23 +32,305 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Update,
     (ActorFunc)EnPart_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Part/EnPart_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Part/EnPart_Destroy.s")
+static Vec3f D_80ACF1B0 = { 0.0f, 0.0f, 0.0f };
+static Vec3f D_80ACF1BC = { 0.0f, 0.0f, 0.0f };
+static Vec3f D_80ACF1C8 = { 0.0f, 0.0f, 0.0f };
+static Vec3f D_80ACF1D4 = { 0.0f, 8.0f, 0.0f };
+static Vec3f D_80ACF1E0 = { 0.0f, -1.5f, 0.0f };
+static Vec3f D_80ACF1EC = { 0.0f, 0.0f, 0.0f };
+static EnPartActionFunc sActionFuncs[] = { func_80ACDDE8, func_80ACE13C, func_80ACE5B8, func_80ACE5C8, func_80ACE7E8 };
+
+extern UNK_PTR D_06001300[];
+extern UNK_PTR D_06001700[];
+extern UNK_PTR D_06001900[];
+extern UNK_PTR D_06001B00[];
+extern UNK_PTR D_06001F00[];
+extern UNK_PTR D_06002100[];
```

Comment:
Given their usage I would think those are textures (whicharrays of UNK_TYPE is better for I think) but I have no idea what object this ends up using so I can't check

(this actor must be the most janky for its object management, which comes from the spawn z rotation in a hardcoded if case in Actor_Spawn)